controllers/memberoperatorconfig/configuration.go

```diff
@@ -28,6 +145,52 @@ func (a MemberStatusConfig) RefreshPeriod() time.Duration {
 	return d
 }
 
+type ToolchainClusterConfig struct {
+	t toolchainv1alpha1.ToolchainClusterConfig
+}
+
+func (a ToolchainClusterConfig) HealthCheckPeriod() time.Duration {
+	defaultClusterHealthCheckPeriod := "10s"
+	healthCheckPeriod := getString(a.t.HealthCheckPeriod, defaultClusterHealthCheckPeriod)
+	d, err := time.ParseDuration(healthCheckPeriod)
+	if err != nil {
+		d, _ = time.ParseDuration(defaultClusterHealthCheckPeriod)
+	}
+	return d
```

Comment:
this could be extracted to a function as well:
```go
func getDuration(value *string, defaultValue string) time.Duration {
	healthCheckPeriod := getString(value, defaultValue)
	d, err := time.ParseDuration(healthCheckPeriod)
	if err != nil {
		d, _ = time.ParseDuration(defaultValue)
	}
	return d
}
```

Reply:
I modified this slightly and added it to the toolchain-common PR: https://github.com/codeready-toolchain/toolchain-common/pull/196
```
func GetDuration(value *string, defaultValue time.Duration) time.Duration {
	durationAsString := GetString(value, "invalid value")
	d, err := time.ParseDuration(durationAsString)
	if err != nil {
		return defaultValue
	}
	return d
}
```